eng/build.sh

```diff
@@ -570,4 +581,19 @@ export DOTNETSDK_ALLOW_TARGETING_PACK_CACHING=0
 cmakeargs="${cmakeargs// /%20}"
 arguments="$arguments /p:TargetArchitecture=$arch /p:BuildArchitecture=$hostArch"
 arguments="$arguments /p:CMakeArgs=\"$cmakeargs\" $extraargs"
+
+if [[ "$bootstrap" == "1" ]]; then
+  # Strip build actions other than -restore and -build from the arguments for the bootstrap build.
+    bootstrapArguments="$arguments"
+    for flag in --sign --publish --pack --test -sign -publish -pack -test; do
+      bootstrapArguments="${bootstrapArguments//$flag/}"
+    done
```

Comment:
nit: indentation is a bit off:
```suggestion
  bootstrapArguments="$arguments"
  for flag in --sign --publish --pack --test -sign -publish -pack -test; do
    bootstrapArguments="${bootstrapArguments//$flag/}"
  done
```